apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -449,7 +445,7 @@ type EnvoyLogging struct {
 	// AccessLogFields sets the fields that JSON logging will
 	// output when AccessLogFormat is json.
 	// +optional
-	AccessLogFields AccessLogFields `json:"jsonFields,omitempty"`
+	AccessLogJSONFields AccessLogJSONFields `json:"accessLogJSONFields,omitempty"`
```

Comment:
rename for consistency

Reply:
I'm OK with this given our stated policy that we can rename things in alpha APIs, let's just make sure we call it out as a breaking change in the release notes.

Reply:
mentioned in release notes!